client/ayon_core/tools/loader/ui/window.py

```diff
@@ -660,3 +764,26 @@ def _on_folders_refresh(self):
 
     def _on_products_refresh(self):
         self._refresh_handler.set_products_refreshed()
+        if (
+            self._auto_select_first_item
+            and not self._selected_version_ids
+            and not self._selected_folder_ids
+        ):
+            self._auto_select_first_item = False
+            self._products_widget.select_latest_item()
```

Comment:
So if I select different version and then hit refresh, it will also change what I had selected in browser?